src/test/java/teammates/test/cases/webapi/DeleteFeedbackSessionActionTest.java

```diff
@@ -23,13 +28,126 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        // see test cases below
+    }
+
+    @Test
+    public void testDeleteFeedbackSessionAction_invalidParameters_shouldThrowHttpParameterException() {
+        CourseAttributes course = typicalBundle.courses.get("typicalCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        ______TS("No course ID");
+        String[] noCourseIdParams = new String[] {
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, session.getFeedbackSessionName(),
+        };
+        verifyHttpParameterFailure(noCourseIdParams);
+
+        ______TS("No session name");
+        String[] noSessionname = new String[] {
+                Const.ParamsNames.COURSE_ID, course.getId(),
+        };
+
+        verifyHttpParameterFailure(noSessionname);
+
+        ______TS("Empty parameters");
+        verifyHttpParameterFailure();
+    }
+
+    @Test
+    public void testDeleteFeedbackSessionAction_typicalCase_shouldPass() throws Exception {
+        ______TS("Delete session that has been soft deleted");
+
+        CourseAttributes course = typicalBundle.courses.get("typicalCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        String[] params = new String[] {
+                Const.ParamsNames.COURSE_ID, course.getId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, session.getFeedbackSessionName(),
+        };
+
+        assertNotNull(logic.getFeedbackSessionDetails(session.getFeedbackSessionName(), course.getId()));
```

Comment:
```suggestion
        assertNotNull(logic.getFeedbackSession(session.getFeedbackSessionName(), course.getId()));
```
Using `getFeedbackSession` is enough?